docs/api-testing/build-from-spec.md

```diff
@@ -2,25 +2,18 @@
 id: build-from-spec
 title: Building an API Test from a Spec File
 sidebar_label: Build Test from Spec File
-description: "The available spec files you can automatically generate from are: Swagger, RAML, OpenAPI 3, API Blueprint, I/O Docs, SOAP WSDL, and Postman Collection."
+description: "Automatically generate tests from an OpenAPI spec or HAR file."
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-export const Highlight = ({children, color}) => ( <span style={{
-      backgroundColor: color,
-      borderRadius: '2px',
-      color: '#fff',
-      padding: '0.2rem',
-    }}>{children}</span> );
-
-Using the API Testing **HTTP Client**, you can generate tests from external endpoints or files such as an OpenAPI Specification (spec) file.
+Using the API Testing **HTTP Client**, you can generate tests from external endpoints or files such as an OpenAPI Specification (spec).
 
 ## What You'll Need
 
 * A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
 * An existing API Testing Project. For details on how to create one, see [API Testing Quickstart](/api-testing/quickstart/)
-* A REST API Specification file in one of our supported formats: <Highlight color="#37b732">Swagger</Highlight> <Highlight color="#4e5b2e">OpenAPI 3</Highlight> <Highlight color="#ff6c37">Postman Collection</Highlight>
+* An OpenAPI Specification file (v3.0 or higher) or HAR (HTTP Archive) file
```

Comment:
@kimsauce even though Postman is not a "spec" file and there's a different page for it, I think we should add it in the list and link the specific page